sql/core/src/test/scala/org/apache/spark/sql/execution/SQLWindowFunctionSuite.scala

```diff
@@ -356,6 +356,42 @@ class SQLWindowFunctionSuite extends QueryTest with SharedSQLContext {
     spark.catalog.dropTempView("nums")
   }
 
+  test("window function: mutiple window expressions specified by range in a single expression") {
+    val nums = sparkContext.parallelize(1 to 10).map(x => (x, x % 2)).toDF("x", "y")
+    nums.createOrReplaceTempView("nums")
```

Comment:
And this test case doesn't cover when CurrentRow is not in the window frame. We'd better add that senario.